### PR TITLE
Update meetups.md

### PR DESCRIPTION
**Pull request by asif-simform**
Added `react-ahmedabad` meetup link in the community.

https://www.meetup.com/react-ahmedabad/

image

### CODE REVIEW

**Comment by gaearon** [comment]
can you please rebase? thanks

**Comment by asif-simform** [comment]
Hi @gaearon 

Resolved conflicts
Thanks

File: content/community/meetups.md
Comment (harish-sethuraman) [inline content/community/meetups.md:1]: I don't think this file is necessary now.

Comment (asif-simform) [reply content/community/meetups.md:1]: Removed that file.

**Review by harish-sethuraman — APPROVED** [review]

**Comment by asif-simform** [comment]
Thanks a lot :)

**Pull request merged**